proactive-jupyter-kernel/pragma.py

```diff
@@ -347,6 +356,10 @@ def is_valid_get_result(data):
     raise ParameterError('Invalid parameters')
 
 
+def is_valid_submitted_jobs(data):
+    pass
```

Comment:
Is it normal it does nothing ?

Reply:
Yes @mbenguig. This is a second level of validation in the pragma parsing process. I used it to validate the structure patterns of the parameters accordingly to all the different way to use a pragma. So this function does nothing because submitted_jobs is a small pragma that does not take any input parameter.